--- a/Config.ini.orig	2026-07-27 09:34:56.866376783 +0000
+++ a/unpacked/gpt.bin.out/2_ROOTFS.bin.out/apps/Config.ini	2026-07-27 09:34:56.878376849 +0000
@@ -16,10 +16,10 @@
 saturation=50
 colorLampMode=0
 language=0
-animationType=0
-idleReturnTime=1000
+animationType=1
+idleReturnTime=500
 vcomVoltage=51
-wallPaper=2.jpg
+wallPaper=3.jpg
 bBackMute=1
 
 videoOutputFormat=0
@@ -68,7 +68,7 @@
 bUseSoftReset=1
 bUseUi1Config=1
 [DEBUG]
-bPrintLog=1
+bPrintLog=0
 bPrintTouch=0
 bPrintThreadCpuLoad=0
 bUseLogBuffer=0
@@ -97,7 +97,7 @@
 startUpDefVolume=20
 startUpMinVolume=5
 startUpMaxVolume=30
-startUpDelayTime=500
+startUpDelayTime=0
 startUpResume=1
 startUpVideoPath=
 startUpMaxTalkVolume=20
